filebeat/fileset/factory.go

```diff
@@ -110,6 +110,7 @@ func (p *prospectorsRunner) Stop() {
 		prospector.Stop()
 	}
 }
-func (p *prospectorsRunner) ID() uint64 {
```

Comment:
Was the ID still used somewhere? If not, perhaps remove it in a different PR?

Reply:
It wasn't used, it's a spurious implementation, probably forgot from a previous removal